docs/reference/cli/dsc.md

```diff
@@ -108,29 +108,47 @@ Type:      String
 Mandatory: false
 ```
 
-### -l, --logging-level
+### -l, --trace-level
 
-Defines the minimum log level DSC should emit during an operation. Messages in DSC are categorized
-by their log level.
+Defines the minimum message level DSC should emit during an operation. Messages in DSC are
+categorized by their level.
 
-The following list shows the valid log levels from highest to lowest level. When this option is
+The following list shows the valid message levels from highest to lowest level. When this option is
 set to any value in the list, DSC emits messages at that level and above.
 
-- `Error`
-- `Warning`
-- `Info` (default)
-- `Debug`
-- `Trace`
+- `error`
+- `warning` (default)
+- `info`
+- `debug`
+- `trace`
```

Comment:
We should mention that `trace` level will write out the contents of JSON input/output and isn't sanitized and only intended for developer use.  As such, such traces should NOT be redirected to storage as it may contain sensitive information.

Reply:
Added the following warning:

> [!WARNING]
> The `trace` level output emits all JSON input/output that DSC processes during execution. DSC doesn't sanitize the JSON before emitting it. This trace level is only intended for developer use. Never redirect `trace` level output to storage as it may contain sensitive information.